src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```
@@ -72,7 +72,7 @@ void CenterOfBrightness::updateState(uint64_t currentSimNanos) {
     if (this->saveImages) {
         dirName = this->saveDir + std::to_string((double)currentSimNanos * NANO2SEC) + ".png";
         if (!cv::imwrite(dirName, imageCV)) {
-            bskLogger.bskLog(BSK_WARNING, "CenterOfBrightness: wasn't able to save images.");
+            throw std::invalid_argument("CenterOfBrightness: wasn't able to save images.");
```

:small_red_triangle_down: Are we sure this should throw an exception?

---

I'm not sure but I don't see why not ..? would you want me to remove it?

---

Because it's not an error worth stopping execution for.

---

Thibaud suggested using this instead:
std::cerr << "Warning: CenterOfBrightness wasn't able to save images." << std::endl;
I tested it by tripping the file path and it worked